docs/release-notes/release-notes-0.19.0.md

```diff
@@ -170,6 +170,10 @@
   range TLVs provided with the existing set of records on the HTLC,
   overwriting any conflicting values with those supplied by the API.
 
+* [Make](https://github.com/lightningnetwork/lnd/pull/9405) the param
```

Comment:
Nit: Separate release notes in own commit

Reply:
updated